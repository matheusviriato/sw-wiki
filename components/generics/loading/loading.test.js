import React from 'react';
import { render } from '@testing-library/react';
import Loading from '.';

describe('Loading tests', () => {
  describe('Render tests', () => {
    it('should match to snapshot', () => {
      const { baseElement } = render(<Loading />);

      expect(baseElement).toMatchSnapshot();
    });
  });
});
