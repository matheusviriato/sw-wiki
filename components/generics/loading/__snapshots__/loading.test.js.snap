// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading tests Render tests should match to snapshot 1`] = `
<body>
  <div>
    <section
      class="css-1mg3v35-loadingWrapper"
    >
      <img
        class="css-1yavjhk-loadingImg"
        src="/img/republic_icon.png"
      />
    </section>
  </div>
</body>
`;
